db.json

```diff
@@ -1,5 +1,35 @@
 {
   "workstreams": [
+    {
+      "id": "browser-testing-api",
+      "name": "Browser Testing API",
+      "scope": "Internal APIs for automating testing of web platform features implemented in web browsers, e.g., to trigger JavaScript garbage collection.",
+      "editors": [
+        {
+          "name": "James Graham",
+          "email": "james@hoppipolla.co.uk"
+        }
+      ],
+      "standards": [
+        {
+          "name": "Browser Testing API",
+          "href": "https://browser-testing-api.spec.whatwg.org/",
+          "description": "The Browser Testing API Standard defines internal APIs for automating testing of web platform features implemented in web browsers.",
+          "authors": [
+            {
+              "name": "James Graham",
+              "email": "james@hoppipolla.co.uk"
+            }
+          ],
+          "reference": "BROWSER-TESTING-API",
+          "review_draft_schedule": [
+            3,
```

Comment:
We had 15 existing standards, with 5 in each of the (6, 12), (1, 7) and (2, 8) buckets. This starts a new (3, 9) bucket.

Reply:
Need to add a calendar entry to not forgot about it when we merge this.